internal/controller/nonadminbackup_controller.go

```diff
@@ -86,31 +87,91 @@ func (r *NonAdminBackupReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		return ctrl.Result{}, err
 	}
 
+	reconcileExit, reconcileErr := r.DeleteVeleroBackup(ctx, rLog, &nab)
```

Comment:
Won't we have a case to requeue here ?

Reply:
Lint job complained that the requeue was not used in the r.DeleteVeleroBackup function, so I removed it from the logic.